### DISCUSSION TITLE
Performance with Radiative Transport

### DISCUSSION THREAD

**Discussion by hwhitehead** [PrincetonUniversity/athena#565] [Q&A] [answered]
I realise this question is likely to be highly specific to the problem at hand, but does anyone have even rough predictions for the additional cost of non-relativistic radiation transport? Obviously there are more equations that need to be solved, but I was wondering how much more intensive it is compared to a simulation with a general equation of state and no magnetic fields.

Best information I could find was in RT code paper ([link](https://iopscience.iop.org/article/10.3847/1538-4357/acc8cf/pdf)) but there the comparisons are made between magnetised relativistic simulations, I am only looking in making the jump from Newtonian hydro to Newtonian hydro+RT.

**Comment by yanfeij** [comment, marked as answer]
You are looking at the wrong code paper. If you want to use the non-relativistic RT, check out the documentation here: https://github.com/PrincetonUniversity/athena/wiki/Non-relativistic-Radiation-Transport

The cost will be very problem dependent and how many angles you need to use. But it can vary from 4 to 10 times the hydro.

**Reply by hwhitehead** [reply]
Thanks, much appreciated!
